pypeit/slittrace.py

```diff
@@ -15,6 +15,7 @@
 from astropy.coordinates import SkyCoord, Angle
 from astropy import units
 from astropy.stats import sigma_clipped_stats
+from numpy.lib.arraysetops import isin
```

Comment:
Is this used?  How is this different from `numpy.isin`?

Reply:
it is not.  removed